errors.py

```
@@ -0,0 +1,3 @@
+class InvalidTestGroupError(Exception):
```

Should we raise a value error instead?
If so, we can also reduce the number of COPY statemetns in Dockerfiles